packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -321,6 +322,17 @@ export enum NodeKind {
     Object = 2
 }
 
+// @public @sealed
+export interface NodeSchemaMetadata<TCustomMetadata = unknown> {
```

Comment:
Interested in thoughts on the type parameter naming here. The more I think about it, the more I'm leaning towards `TUserMetadata`.